#4dpzbk / - #2bj0dv - Add meta tags (#395)
pupil-labs/pupil-docs: src/core/terminology.md

---
permalink: /core/terminology

> description: In this section, we will introduce a few key terms/concepts that are essential to Pupil Core.

Reviewer: ```suggestion
description: Key terms and concepts that are essential to Pupil Core.
```
